charts/app-config/values-integration.yaml

```diff
@@ -424,6 +424,9 @@ govukApplications:
             key: SECRET_KEY_BASE
       - name: REDIS_URL
         value: redis://shared-redis-govuk.eks.integration.govuk-internal.digital
+ - name: draft-collections
+   helmValues:
+     <<: *collections
```

Comment:
Are you sure there isn't anything that needs setting differently for the draft version? (there must be, surely?)

Reply:
looking at blue-frontend and blue-draft-frontend EC2 - i dont really see a difference other then maybe no need to use Memcache

Reply:
```diff
$ diff <(sshi frontend:1 -- govuk_setenv collections env | sort) <(sshi draft_frontend:1 -- govuk_setenv collections env | sort)
29d28
< MEMCACHE_SERVERS=frontend-memcached:11211
31a31,37
> PLEK_HOSTNAME_PREFIX=draft-
> PLEK_SERVICE_ACCOUNT_API_URI=https://account-api.integration.publishing.service.gov.uk
> PLEK_SERVICE_IMMINENCE_URI=https://imminence.integration.publishing.service.gov.uk
> PLEK_SERVICE_LICENSIFY_URI=https://licensify.integration.govuk-internal.digital
> PLEK_SERVICE_LOCAL_LINKS_MANAGER_URI=https://local-links-manager.integration.publishing.service.gov.uk
> PLEK_SERVICE_SEARCH_API_URI=https://search-api.integration.govuk-internal.digital
> PLEK_SERVICE_SEARCH_URI=https://search.integration.govuk-internal.digital
```

Reply:
Memcache matters as much as Plek stuff btw — if we were to accidentally hook up draft-collections to memcache, we could easily expose pre-publication content on the public website.

Reply:
You can copy from another draft app for the Plek stuff — you probably only need to set `PLEK_HOSTNAME_PREFIX` and the global settings from env-configmap should take care of the overrides for you.

Reply:
Great - i just compared env vars rather then the Govuk_setenv - thanks updated